src/Components/CrypticChar.js

```diff
@@ -2,7 +2,7 @@ import { useEffect, useState } from "react";
 import "./CrypticChar.css"
 
 const timeout = 100;
-const crytpicChars = ['~', '!', '@', '#', '$', '%', '^', '&', '*', '(', ')', '+', '=', '-', '[', ']', '{', '}', '<', '>', '?', '/', ';', ':']
+const crytpicChars = ['~', '!', '@', '#', '$', '%', '^', '&', '*', '(', ')', '+', '=', '-', '[', ']', '{', '}', '<', '>', '?', '/', ';', ':','♬','♣','☻','§','♀','♂','☎','☂','◊','❦','♝']
```

Comment:
Id prefer the more normal utf characters, these are more emoji-like. If you want to switch them to be less emoji like I'd be happy to merge!